### ISSUE TITLE
voiceover does not announce rewards when checking off a task

### ISSUE THREAD

**Issue by rilygreat** [HabitRPG/habitica-android#2188]
> iOS Version: 26.3
>  Device: iPhone17,4
>  App Version: 4.0 (1048)
>  User ID: c29f3ff4-2a5c-428f-8802-49f4e2ec93f3
> 
> There seems to be some accessibility issues with the app. When you check off a habit or another task, voiceover does not announce to blind users the rewards they gained, especially if we have found an item, the only way I know how much gold I got is by looking at the gold bar, it doesn’t announce how much gold I get.There should be louder, more pronounced sound effects when checking off a task. this would not only help blind users know when we have successfully checked something off, but I think it would also make the app more fun.
